### PR TITLE
fix ui common data - add layer

### PR DESCRIPTION
**Pull request by piensaenpixel**
@javierarce  could you take a look?

image

### CODE REVIEW

**Comment by xavijam** [comment]
:+1:

File: lib/assets/javascripts/cartodb3/components/modals/add-layer/content/datasets/remote-dataset-item-view.js [hunk 1]
```diff
@@ -68,11 +68,7 @@ module.exports = DatasetItem.extend({
 
   _setItemClasses: function () {
     // Item selected?
-    this.$el.toggleClass('is--selected', !!this.model.get('selected'));
-    // Check if it is selectable
-    this.$el.toggleClass('DatasetsList-item--selectable', !!this._canImportDataset());
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/components/modals/add-layer/content/datasets/remote-dataset-item-view.js]: Hey, why have you removed those things? They are important to know if the user can import or not the dataset. Those states are a must please.

**Comment by xavijam** [comment]
Check my comment in the code, they should be fixed before merge please.

**Comment by piensaenpixel** [comment]
fixed @xavijam you're right, sorry i deleted when change the className.

**Comment by javierarce** [comment]
👍

**Pull request merged**